lib/IAMClient.js

```diff
@@ -356,8 +398,12 @@ class IAMClient {
      * @returns {undefined}
      */
     request(method, path, callback, data, reqUid) {
+        let httpMethod = method;
+        if (method === 'IAMPOST' || method === 'S3POST') {
```

Comment:
Same comment here as in scality/Vault#612  We should not be using this method to address a slight variation in the way we manage the protocol. It would be much better to stop one minute, observe carefully our needs, and design an adapted solution. Please discuss this with @alexandre-merle as the harmonization and cleanup of a protocol took much of his time, and we should stay along the lines we drew up.